packages/dicomImageLoader/src/imageLoader/wadors/metaData/getNumberValue.ts

```diff
@@ -1,13 +1,17 @@
-import getValue from './getValue.js';
+import { WADORSMetaDataElement } from '../../../types';
+import getValue from './getValue';
 
-function getNumberValue(element, index) {
+function getNumberValue(
+  element: WADORSMetaDataElement,
+  index?: number
+): number {
   const value = getValue(element, index);
 
   if (value === undefined) {
     return;
   }
 
-  return parseFloat(value);
+  return parseFloat(value as string);
```

Comment:
This isn't a good assumption here - parseFloat will handle string, but also will handle number natively, and some of the big number formats.  Just let typescript complain about it if it isn't happy.